<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>881</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>All Musics</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>881</width>
        <height>391</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Album</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Length</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Edit</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-110</y>
        <width>345</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>SPOTIMAC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Playlist reader</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetPlaylist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>881</width>
        <height>351</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Album</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Length</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Edit</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Open your file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonM3U">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="album">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="artist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="positionSlider">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>311</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="track">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>80</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>80</width>
      <height>33</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>460</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>80</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="length">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="position">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>positionSlider</zorder>
   <zorder>tabWidget</zorder>
   <zorder>album</zorder>
   <zorder>artist</zorder>
   <zorder>track</zorder>
   <zorder>nextButton</zorder>
   <zorder>pushButton</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>prevButton</zorder>
   <zorder>length</zorder>
   <zorder>position</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
